lib/std/debug.zig

```diff
@@ -1483,8 +1489,26 @@ fn dumpSegfaultInfoPosix(sig: i32, code: i32, addr: usize, ctx_ptr: ?*anyopaque)
         .aarch64,
         .aarch64_be,
         => {
-            const ctx: *posix.ucontext_t = @ptrCast(@alignCast(ctx_ptr));
-            dumpStackTraceFromBase(ctx);
+            // Some kernels don't align `ctx_ptr` properly. Handle this defensively.
+            const ctx: *align(1) posix.ucontext_t = @ptrCast(ctx_ptr);
+            var new_ctx: posix.ucontext_t = ctx.*;
+            if (builtin.os.tag == .macos and builtin.cpu.arch == .aarch64) {
```

Comment:
```suggestion
            if (builtin.os.tag.isDarwin() and builtin.cpu.arch == .aarch64) {
```

Oops, should have used this.

Reply:
Sure, changed.